common/util.c

```diff
@@ -253,7 +253,7 @@ v_wstrdup(SCR *sp, const CHAR_T *str, size_t len)
 	if (copy == NULL)
 		return (NULL);
 	MEMCPY(copy, str, len);
-	copy[len] = L('\0');
+	copy[len] = '\0';
```

Comment:
This was an unnecessary change on my part, so I reverted it.

Reply:
I actually prefer the way in #96. But for foolish consistency, sure...